src/testRunner/unittests/tsserverProjectSystem.ts

```diff
@@ -10667,7 +10671,7 @@ declare class TestLib {
             });
         });
 
-        it("can go to definition correctly", () => {

```

Comment:
Sidebar: can we break down the general `tsserverProjectSystem` unnittest file into like 10 more specific subfiles? It's over 10000 lines long now, which is not great. A lot of it would just be breaking certain `describe` blocks into their own files.

Reply:
Will do that but in separate PR so as to avoid unintentional conflicts and merge issues.

Reply:
Working PR https://github.com/Microsoft/TypeScript/pull/28902 for this. But will complete that only after merging this change to avoid merge conflicts.